great_expectations_provider/example_dags/example_great_expectations_dag.py

```diff
@@ -24,13 +23,10 @@
 )
 
 base_path = Path(__file__).parents[2]
-data_dir = os.path.join(base_path, "include", "data")
-data_file = os.path.join(
-    data_dir,
-    "yellow_tripdata_sample_2019-01.csv",
-)
+data_dir = base_path / "include" / "data"
+data_file = data_dir / "yellow_tripdata_sample_2019-01.csv"
 
-ge_root_dir = os.path.join(base_path, "include", "great_expectations")
+ge_root_dir = base_path / "include" / "great_expectations"
```

Comment:
Not entirely sure how base_path works, but are there not +'s missing here?

Reply:
No. The `/` operator can be used to join paths. 

https://docs.python.org/3/library/pathlib.html#operators

So the following will all produce the same path object.
```python
ge_root_dir = base_path / "include" / "great_expectations"
ge_root_dir = base_path.joinpath("include" , "great_expectations")
ge_root_dir = Path(base_path, "include" , "great_expectations")
```

However, because I want to convert to a str at the end I'm going to use one of the later 2 so I can do it in one line.

```python
ge_root_dir: str = str(base_path.joinpath("include" , "great_expectations"))

# compared to ...
ge_root_dir = base_path / "include" / "great_expectations"
ge_root_dir = str(ge_root_dir)
```

Reply:
@denimalpaca updated.
Also here's another good article explaining `pathlib`

https://realpython.com/python-pathlib

Reply:
Thanks for the links, looking into that.